### PR TITLE
Dev aaron

### PR DESCRIPTION
**Pull request by ayobi**

### CODE REVIEW

File: pephub/routers/api/v1/project.py [hunk 1]
```diff
@@ -151,20 +139,17 @@ async def update_a_pep(
...
+
+        try:
+            # validate project (it will also validate samples)
+            eido.validate_project(
+                new_project, "http://schema.databio.org/pep/2.1.0.yaml"
+            )
+        except Exception as e:
+            raise HTTPException(
+                status_code=400,
+                detail=f"",
```
Comment (nleroy917) [inline pephub/routers/api/v1/project.py:151]: ? no detail

File: web/src/hooks/mutations/useTotalProjectChangeMutation.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  const queryClient = useQueryClient();
+
+  const mutation = useMutation({
+    mutationFn: () => editTotalProject(namespace || '', project || '', tag, jwt || '', data),
+    onSuccess: () => {
+      queryClient.invalidateQueries([namespace, project, tag]);
+      toast.success('Successfully updated the project!');
+    },
+    onError: (error: AxiosError) => {
+      toast.error(`Failed to update project: ${error}`);
```
Comment (nleroy917) [inline web/src/hooks/mutations/useTotalProjectChangeMutation.ts:30]: Maybe in the near future, we can add proper error handling

**Review by nleroy917 — APPROVED** [review]
Looks good - there might be areas we can optimize in the future, but for now this is good to solve #207

**Comment by ayobi** [comment]
True, I'll address that and others with #164

File: pephub/routers/api/v1/project.py (hunk 1 above)
Comment (ayobi) [reply pephub/routers/api/v1/project.py:151]: note sure about this from @khoroshevskyi

**Pull request merged**